src/fragments/start/getting-started/flutter/build.mdx

```diff
@@ -1,8 +1,15 @@
 ## What we'll build
 
+The stable release of Amplify Flutter currently supports iOS/Android as target platforms.
+
 <Callout warning>
 
-Amplify Flutter currently supports iOS/Android as target platforms. Flutter [Web](https://github.com/aws-amplify/amplify-flutter/issues/234) and [Desktop](https://github.com/aws-amplify/amplify-flutter/issues/133) support are feature requests that are being tracked in our Github repo.
+Web and desktop support for the Amplify Flutter Auth category is now in developer preview! 
```

Comment:
This language is repeated elsewhere in this PR - should probably be pulled out

Reply:
This is specific to the 'getting started' docs section, which currently offer a datastore-only tutorial. I wanted to get something into this section highlighting the fact that a) web/desktop support is availbable, and b) it isn't used in the getting started tutorial.

I'm doing a second PR after this one to build in Auth into the getting started, and this will have switchers for dev preview. So this text is going to get updated eventually. I didn't want to block the rest of the docs updates on the tutorial update.